tools/mason/MasonNew.chpl

```diff
@@ -47,18 +50,40 @@ proc masonNew(args) throws {
         else if arg == '--show' {
           show = true;
         }
+        else if arg.startsWith('--name') {
+          if arg.startsWith('--name=') {
+            var res = arg.split("=");
+            packageName = res[2];
+            dirName = args[2];
+          }
+          else {
+            packageName = args[countArgs];
+            dirName = args[2];
+          }
+        }
         else {
-          name = arg;
+          if packageName.length == 0 then {
+            dirName = arg;
+            packageName = dirName;
+          }
+        }
+      }
+      if dirName != packageName then {
```

Comment:
We shouldn't need to check if they are equal. If they are then, the calls are the same.

Reply:
Let's add a newline here to visually separate this from the arg parsing